src/ghost-content/index.js

```diff
@@ -4,12 +4,18 @@ import Promise from 'bluebird'
 import glob from 'glob'
 import uuid from 'uuid'
 import get from 'lodash/get'
+import partition from 'lodash/partition'
```

Comment:
Would `import { get, partition } from lodash` work?

Reply:
Hm, sent the reply over email but don't see it here. It will work but I've seen tree shaking not always working correctly in such case, individual imports are more reliable

Reply:
Ah, it was hanging unattached to the thread.